packages/node-sdk/paima-db/migrations/up.sql

```diff
@@ -17,7 +17,13 @@ CREATE TABLE scheduled_data_tx_hash (
 
 CREATE TABLE scheduled_data_extension (
   id INTEGER PRIMARY KEY REFERENCES scheduled_data(id) ON DELETE CASCADE,
-  cde_name TEXT NOT NULL
+  cde_name TEXT NOT NULL,
+  network TEXT NOT NULL
+);
+
+CREATE TABLE scheduled_data_precompile (
```

Comment:
I'm not really sure it makes sense to have this also as a new table instead of just adding a column tbh.

Reply:
precompiles are not just scheduled data. They're for anything where you need to assign an address to something that isn't a real player (ex: buying from a shop could be considered sending funds to a precompile address)

Reply:
I'm not sure what that means in relation to this code. This is just to associate a particular scheduled input with a particular precompile in order to compute the hash. 

I'm not really storing the precompiles by themselves in the db, those are just imported from the packaged/precompiles.cjs and injected to the engine on launch.

Reply:
And the reason this is needed is because when you call `createScheduledData` you pass the precompile 'name' which is used as the address when the event is triggered.